tools/cockpit.spec

```diff
@@ -444,12 +429,40 @@ fi
 %systemd_preun cockpit.socket cockpit.service
 
 %postun ws
-if [ -x %{_sbindir}/selinuxenabled ]; then
-    %selinux_modules_uninstall -s %{selinuxtype} %{name}
-    %selinux_relabel_post -s %{selinuxtype}
-fi
 %systemd_postun_with_restart cockpit.socket cockpit.service
 
+%package selinux-policies
```

Comment:
How about a middle ground and calling this `cockpit-ws-selinux`? That makes it conformant with all the other myriad *-selinux packages?

Reply:
This works for me too.

Reply:
Renamed to cockpit-ws-selinux.